src/test/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaDataTest.java

```diff
@@ -186,6 +186,17 @@ void testGetTables() throws SQLException {
         }
     }
 
+    /**
+     * Test getTables returns empty ReseultSet when empty string schemaPattern is passed.
```

Comment:
typo

Reply:
could you let me know where is the typo please?

Reply:
ReseultSet

Reply:
I've fixed the typos.